packages/editor/src/components/properties/ModelNodeEditor.tsx

```diff
@@ -153,20 +154,13 @@ export const ModelNodeEditor: EditorComponentType = (props) => {
           onChange={updateProperty(ModelComponent, 'avoidCameraOcclusion')}
         />
       </InputGroup>
-      <InputGroup name="Loop Animation" label={t('editor:properties.model.lbl-loopAnimation')}>
-        <SelectInput
-          key={props.entity}
-          options={animationOptions.value}
-          value={loopAnimationComponent?.activeClipIndex}
-          onChange={onChangePlayingAnimation}
-        />
-      </InputGroup>
       <InputGroup name="Is Avatar" label={t('editor:properties.model.lbl-isAvatar')}>
```

Comment:
Is avatar belongs to loop animation component too

Reply:
yup thats why i asked, but we still need to reference it in model node editor right?

Reply:
added to PrefabEditors.tsx, made it reactive and moved is avatar as well